### ISSUE TITLE
Tanzania - GIS and Clinical Cascade - Maps and Analytic Support 

### ISSUE THREAD

**Issue by hessionra** [ICPI/COP18#86]
Request submitted by SI Advisor Nadia
Approved by ICPI
Assigned to Imran and Dustyn with Josh facilitating

**Background:**
Nadia Solehdin forwarded the following message from the TZ team (in particular Kevin Cain, the CDC Country Director)on 2/26.

Team Tanzania would like to request assistance in slide development from the ICPI team here tomorrow and Wednesday, in preparation for our outbrief presentation. 
We plan to storyboard our presentation, slot in existing slides, and determine what additional slides are needed. We would need help with potentially modifying what we have and definitely creating the new.
Ritu introduced me to the team at their table today, and they asked that we request through you.
Thanks,
Kevin

**Additional Information**

**Next Steps:**
The best next step would be for a couple of us to sit with you to discuss our overall plan for the presentation, get your feedback, and start working on anything that we decide immediately that we need. Can we meet sometime this morning, perhaps after the coffee break? 

**Ideas for Maps** @dp1984 
For starters, I think having some map that show our treatment gaps would be helpful. A map showing ART treatment gap by region and council would help. Those by sex would also help. It may be useful to do some age disaggs within sex as well, but I’d start with the above general categories.

My thought is that these would serve as a base slide on which we could overlay things like the % of pos that are coming from index testing, the % of tests coming from community, etc.

**Comment by jaliasd** [comment]
Nadia said the request was from Kevin K the CDC country director and to follow up with him.

**Comment by jb3436** [comment]
Additional Information> For starters, I think having some map that show our treatment gaps would be helpful. A map showing ART treatment gap by region and council would help. Those by sex would also help. It may be useful to do some age disaggs within sex as well, but I’d start with the above general categories.

My thought is that these would serve as a base slide on which we could overlay things like the % of pos that are coming from index testing, the % of tests coming from community, etc.

Kevin
Sent from my iPhone

On Feb 27, 2018, at 9:05 AM, Cain, Kevin (CDC/CGH/DGHT) <bvz1@cdc.gov> wrote:
Thanks! I think the best next step would be for a couple of us to sit with you to discuss our overall plan for the presentation, get your feedback, and start working on anything that we decide immediately that we need. Can we meet sometime this morning, perhaps after the coffee break?

**Comment by jb3436** [comment]
Imran and Dustyn are working on the request based on the informaiton we have. Dustyn is creating several maps and Imran is creating tables of several scenarios. @dp1984 @imujawar , feel free to add additional details.

We will continue to look for a time to meet with the Tanzania team.

**Comment by dp1984** [comment]
Imran and I just discussed with Kevin on the requests for Tanzania. He is going to provide spectrum data, intervention data, and data for coverage table (raster) for all the maps and visuals. 

1. map and visuals for overall treatment gap map as well as above 
2. map and visuals for above & below 15/male & female treatment gap maps. 
3. map and visuals for what % of positives coming from index testing for men above 15
4. projections of hts pos based on interventions (pitc, index testing, community testing). 
5. coverage (raster) tables by region

**Comment by imujawar** [comment]
Received data from Jenny Ward: 
1. PLHIV and Treatment data by fine age/sex bands for 193 PSNUs (31 Regions: 26 mainland + 5 Zanzibar).  The StopLight/Heat tables for coverage are at the PSNU level, and need to be aggregated up to Region. 
2. Shape files for the updated PSNUs and regions. 

Follow-up items:
- Will receive HTS_TST_POS projections tomorrow for the projection visualizations. 
- Need to confirm whether the Gap maps for Males 15+ will show pie charts (pacman) charts for % positives coming from Index testing.

**Comment by ICPIadmin** [comment]
Assigning @MicSherlock who is available for QC.

**Comment by imujawar** [comment]
Clarity on the HTS_POS request:
- FY17 QUARTERLY TRENDS in % pos contribution from community, PITC, Index. Total. 
- Overall
- 27 regions (matrix). 
- Adding Age-Sex slicers if time permits.

**Comment by dp1984** [comment]
sent Kevin & Jenny maps for 1, 2 as well as treatment gap for community and facility testing for males age 15+

will work on 3 tomorrow. Kevin is ok with this.

**Comment by dp1984** [comment]
We've sent updated slides w/ treatment gap maps (relative & absolute) for overall #'s, as well as male/female (age 15+), male (15+) w/ proportion of pos found and facility and community, and male+females under 15. We also included visualization of overall hts_tst_pos trends by % of index, pitc, and community. Also included in this ppt are regional modality trends (27 regions). 

These slides are located [here ](https://www.pepfar.net/OGAC-HQ/icpi/Shared%20Documents/COP%20Support/COP18/Support%20Requests%20and%20Tracker/GIS/Tanzania/Tanzania_req_TxGap_Modality_trends_20180228.pptx).

We've had to revise maps and visuals a few times as we were using data provided by country team and there were errors in the data as well as updates. 

@imujawar is currently working on intervention strategy prevention projections based on data provided by Kevin Cain. Sending very soon!
